example/hello_classification_electron/ui.js

```diff
@@ -16,26 +18,26 @@ if (!location.search) {
  */
 
 $(document).ready(() => {
-    $('#img').click(() => {
-      us = 'image';
-      if (currentBackend && currentDevice && currentModel) {
-        strsearch = `?b=${currentBackend}&plugin=${currentDevice}&m=${
-            currentModel}&s=${us}`;
-      } else {
-        strsearch = `?b=${ub}&plugin=${up}&m=${um}&s=${us}`;
-      }
+  $('#img').click(() => {
+    us = 'image';
+    if (currentBackend && currentDevice && currentModel) {
+      strsearch = `?b=${currentBackend}&plugin=${currentDevice}&m=${
+          currentModel}&s=${us}`;
+    } else {
+      strsearch = `?b=${ub}&plugin=${up}&m=${um}&s=${us}`;
+    }
 
-      window.history.pushState(null, null, strsearch);
-      updateScenario();
-    });
+    window.history.pushState(null, null, strsearch);
+    updateScenario();
+  });
 
-    $('#cam').click(() => {
-      us = 'camera';
-      strsearch = `?s=${us}`;
-      window.history.pushState(null, null, strsearch);
-      updateScenario(true);
-    });
+  $('#cam').click(() => {
+    us = 'camera';
+    strsearch = `?s=${us}`;
+    window.history.pushState(null, null, strsearch);
+    updateScenario(true);
```

Comment:
Are above lines only for coding style change? If they are, please group them into a separated commit to fix coding style.